pyvista/core/filters/poly_data.py

```diff
@@ -1592,28 +1600,35 @@ def decimate(
             volume preservation is disabled and if ``attribute_error``
             is active, these errors can be large.
 
+        enable_all_attribute_error: bool, default: False
```

Comment:
nits The argument order in the function definition does not match the order in the docstring. Please correct this discrepancy.